### ISSUE TITLE
add support for extensions: 1=lighting, 2=water

### ISSUE THREAD

**Issue by bertt** [bertt/quantized-mesh-tile-cs#3]
add support for extensions: 1=lighting, 2=water

**Comment by polluxll** [comment]
hi, I want to know how to generate the terrain file,Is  there any tools?

**Comment by bertt** [comment]
yes take a look at https://github.com/bertt/awesome-quantized-mesh-tiles

**Comment by bertt** [comment]
sample request with water and lightning extensions for cesium qm tiles:
https://assets.cesium.com/1/10/1077/749.terrain?extensions=octvertexnormals-watermask&v=1.1.0

**Comment by SHIHUAMarryMe** [comment]
Hi, your codes spark me! 
will support to write terrain in future?
